atlasdb-service/src/main/java/com/palantir/atlasdb/impl/AtlasDbServiceImpl.java

```diff
@@ -175,7 +175,7 @@ public Void execute(Transaction t) {
 
     @Override
     public void truncateTable(final String fullyQualifiedTableName) {
-        kvs.truncateTable(getTableRef(fullyQualifiedTableName));
+        kvs.truncateTable(TableReference.createFromFullyQualifiedName(fullyQualifiedTableName));
```

Comment:
Don't think you can switch the one in the case a system table is passed in here.  Say '_locks*'